dataframely/_base_schema.py

```diff
@@ -171,7 +171,11 @@ def column_names(cls) -> list[str]:
     @classmethod
     def columns(cls) -> dict[str, Column]:
         """The column definitions of this schema."""
-        return getattr(cls, _COLUMN_ATTR)
+        columns: dict[str, Column] = getattr(cls, _COLUMN_ATTR)
+        for name in columns.keys():
+            # Dynamically set the name of the columns.
+            columns[name]._name = columns[name].alias or name
```

Comment:
Why is this not
```suggestion
            columns[name]._name = name
```
? Shouldn't `columns` already have the "true" column names?